x/emissions/keeper/queryserver/query_server_losses.go

```diff
@@ -61,3 +61,29 @@ func (qs queryServer) GetReputerLossBundlesAtBlock(ctx context.Context, req *typ
 
 	return &types.GetReputerLossBundlesAtBlockResponse{LossBundles: reputerLossBundles}, nil
 }
+
+func (qs queryServer) GetOpenReputerSubmissionWindows(ctx context.Context, req *types.GetOpenReputerSubmissionWindowsRequest) (_ *types.GetOpenReputerSubmissionWindowsResponse, err error) {
```

Comment:
this feels like a strange file (based on name) to add these to.  normally i would call this a nitpick, but given that we're trying to provide a good foundation (not just in terms of architecture but also semantics, so to make the codebase more easily navigable) i think it's not just a nitpick